kingdon/multivector.py

```diff
@@ -18,6 +19,14 @@ class MultiVector:
     _values: list = field(default_factory=list)
     _keys: tuple = field(default_factory=tuple)
 
+    def copy(self):
+        return self.algebra.multivector(keys=self._keys[:], values=self._values[:])
```

Comment:
I would use self.fromkeysvalues(...) to construct a new MV because that skips all the sanity checks and is therefore more performant. https://kingdon.readthedocs.io/en/stable/module_docs.html#kingdon.multivector.MultiVector.fromkeysvalues.

Also, for a shallow copy shouldn't we use the same keys and values as in the original? Only the MV class should be copied, not the other objects it makes reference to.